syntax = "proto2";

package opentxs.proto;
option optimize_for = CODE_SIZE;

import public "NymIDSource.proto";
import public "SourceProof.proto";

message MasterCredentialParameters {
    optional uint32 version = 1;
    optional nymIDSource source = 2;
    optional sourceProof sourceProof = 3;
}
